server/server.ts

```diff
@@ -60,4 +66,34 @@ server.post('/create', async (req, res) => {
   }
 });
 
+server.post("/join", async (req, res) => {
```

Comment:
I was looking back at the readme and other discussions.  Would it make more sense for this to be /room/join?
Totally open for any suggestions ro quests.

Reply:
Thanks Josh.  Updated.